src/Mafia/Script.hs

```diff
@@ -5,11 +5,20 @@ module Mafia.Script (
     ScriptError(..)
   , renderScriptError
   , runScript
+
+  -- For testing.
+  , Pragma (..)
```

Comment:
This whole script change seems unrelated to building in travis. Should it be it's own PR?